lib/Value/Matrix.pm

```diff
@@ -837,29 +837,117 @@ sub twiddle {
 	return $self->make(@coords);
 }
 
+=head3 C<slice>
+
+Produce the degree (n-1) Matrix defined by a given index and value for that index. If n is 1,
+this produces a Real/Complex/Fraction.
```

Comment:
I think this could use some clarification as to what this really means.  What do `index` and `value` mean in this context?

Reply:
I tried to do better here. I will push and you can see. Imagine a 5x7x9 matrix. Entry positions are defined by three indices i,j,k. The first index, second index, and third index. So "index" here would be 1, 2, or 3. Say it is 3. Then a value for that index can be 1, 2, ..., or 9. Take a look and see if what I changed to is OK or can be improved upon.

Reply:
I think what you changed it to is better.  What you said here really is what really worked for me though.  Perhaps adding an example like that to the POD would better.  Terminology with these "tensor" things can get confusing!